src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1581,23 +1586,28 @@
 
 helps['webapp identity assign'] = """
 type: command
-short-summary: assign or disable managed service identity to the web app
+short-summary: assign managed service identity to the web app
 examples:
   - name: assign local identity and assign a reader role to the current resource group.
     text: >
         az webapp identity assign -g MyResourceGroup -n MyUniqueApp --role reader --scope /subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/MyResourceGroup
```

Comment:
Is this example still accurate? is the identity defaulted to system-assigned?

Reply:
Yes, default is system-assigned.